sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -239,6 +239,7 @@ async def open_async(self, connection=None):
         # pylint: disable=protected-access
         if self._session:
             return  # already open.
+        _logger.info("Opening AMQP client %r.", self._name)
```

Comment:
would it be good to have a session already open in the if statement above?

Reply:
This would log for every operation then which imo I think would clutter the logs?